README.md

```diff
@@ -15,37 +15,19 @@
     ·
     <a href="https://github.com/TheNerdful8/Lecture2Gether/pulls">Send a Pull Request</a>
   </p>
+  <p align="center">
+    <img src="https://github.com/TheNerdful8/Lecture2Gether/blob/master/static/L2Ge_frontpage.png" alt="Screenshot Frontpage" height="200">
+    <img src="https://github.com/TheNerdful8/Lecture2Gether/blob/master/static/L2Ge_player.png" alt="Screenshot Player" height="200">
+  </p>
 </p>
 
 ## About The Project
-<!-- TODO add screenshot -->
 
-Lecture2Gether makes it possible to watch online lectures with friends by pasting a link to a [Lecture2Go](https://github.com/lecture2go/portal-6.2-ce-ga6) Video, a YouTube Video or a simple mp4 link.
+Lecture2Gether makes it possible to watch online lectures with friends by pasting a link to a [Lecture2Go](https://github.com/lecture2go/portal-6.2-ce-ga6) Video, a YouTube or Google Drive Video or a simple .mp4 link.
```

Comment:
Maybe something like "Lecture2Go, YouTube or Google Drive Video". Listing the possible players and only saying video 2 of 3 times seems to be correct, but sounds a bit odd in this context.